rubocop/.gitignore

```diff
@@ -0,0 +1,12 @@
+/.bundle/
+/.yardoc
+/Gemfile.lock
```

Comment:
Gemfile.lock is ignored by default. The same story [here](https://github.com/fiscal-cliff/sequel-tablefunc/blob/master/.gitignore) for instance